src/DrawerLayout.js

```diff
@@ -212,7 +212,7 @@ export default class DrawerLayout extends Component {
   }
 
   _shouldSetPanResponder = (e, { moveX, dx, dy }) => {
-    if (!dx || !dy) {
+    if (!dx || !dy || Math.abs(dx) < 3) {
```

Comment:
I believe you are correct. If we go off of Dimensions.get("window").width, it will not take in account if you rotated the screen which would get pretty ugly code wise...  We have about 5 Android devices on our team and a couple tablets.  If gitlab ever gets done with its nap... sigh... we will see how it works on all our devices.